CHANGELOG.md

```diff
@@ -1,19 +1,15 @@
-# NEXT RELEASE
+# 10.5.4 Release notes
```

Comment:
All bugs also related to v6 should be backported. If not done immediately, please create issue for it.

Reply:
https://github.com/realm/realm-core/issues/4495